brig/cmd/brig/commands/project_create.go

```diff
@@ -0,0 +1,535 @@
+	projectCreateOut         string
+	projectCreateFromProject string
+	projectCreateDryRun      bool
+	projectCreateNoPrompts   bool
+	projectCreateReplace     bool
+)
+
+var defaultProject = &brigade.Project{
+	Name: "deis/empty-testbed",
+	Repo: brigade.Repo{
+		Name:     "github.com/deis/empty-testbed",
+		CloneURL: "https://github.com/deis/empty-testbed.git",
+		SSHKey:   "",
+	},
+	Secrets:      map[string]string{},
+	SharedSecret: "",
+	Github:       brigade.Github{},
+	Kubernetes:   brigade.Kubernetes{},
+	Worker: brigade.WorkerConfig{
+		PullPolicy: "IfNotPresent",
+	},
+}
+
+func init() {
+	project.AddCommand(projectCreate)
+	flags := projectCreate.Flags()
+	flags.StringVarP(&projectCreateConfig, "config", "f", "", "Path to JSON Kubernetes Secret.")
+	flags.StringVarP(&projectCreateFromProject, "from-project", "p", "", "Retrieve the given project from Kubernetes and use it to set the default values.")
+	flags.StringVarP(&projectCreateOut, "out", "o", "", "File where configuration should be saved. The configuration is stored as a JSON Kubernetes Secret")
+	flags.BoolVarP(&projectCreateDryRun, "dry-run", "D", false, "Do not send the config to Kubernetes")
+	flags.BoolVarP(&projectCreateNoPrompts, "no-prompts", "x", false, "Do not prompt the user for input. Use this with -f to skip prompts.")
+	flags.BoolVarP(&projectCreateReplace, "replace", "r", false, "Replace an existing project with this revised one. Use this to modify existing projects.")
+}
+
+var projectCreate = &cobra.Command{
+	Use:   "create",
+	Short: "create a new Brigade project",
+	Long:  projectCreateUsage,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if projectCreateFromProject != "" && projectCreateConfig != "" {
+			return errors.New("cannot specify both -f/--config and -p/--from-project")
+		}
+		return createProject(cmd.OutOrStderr())
+	},
+}
+
+func createProject(out io.Writer) error {
+	c, err := kubeClient()
+	if err != nil {
+		return err
+	}
+
+	store := kube.New(c, globalNamespace)
+
+	proj := defaultProject
+	if projectCreateConfig != "" {
+		if proj, err = loadProjectConfig(projectCreateConfig, proj); err != nil {
+			return err
+		}
+	}
+
+	if projectCreateFromProject != "" {
+		var err error
+		proj, err = store.GetProject(projectCreateFromProject)
+		if err != nil {
+			return fmt.Errorf("could not load %s: %s", projectCreateFromProject, err)
+		}
+	}
+
+	if !projectCreateNoPrompts {
+		if err := projectCreatePrompts(proj); err != nil {
+			return err
+		}
+	}
+
+	secret, err := kube.SecretFromProject(proj)
+	if err != nil {
+		return err
+	}
+	data, err := json.MarshalIndent(secret, "", "  ")
+	if err != nil {
+		return err
+	}
+
+	if globalVerbose {
+		fmt.Printf("%s\n", data)
+	}
+
+	if projectCreateOut != "" {
+		if err := ioutil.WriteFile(projectCreateOut, data, 0755); err != nil {
+			return err
+		}
+	}
+
+	fmt.Fprintf(out, "Project ID: %s\n", proj.ID)
+	if projectCreateDryRun {
+		return nil
+	}
+
+	if !projectCreateReplace {
```

Comment:
I think I would opt for doing that as a separate PR. The `--replace` will walk you through the questions again, but with the old value s pre-populated. I figured this would be useful for people who needed help text and stuff like that.